spec.emu

```diff
@@ -71,7 +71,7 @@ contributors: Google, Ecma International
       <emu-alg>
         1. If _type_ is `"language"`, then
           1. If _code_ does not matches the `unicode_language_id` production, throw a *RangeError* exceptipon.
-          1. Let _code_ be the result of mapping _code_ to lower case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
+          1. Set _code_ to CanonicalizeUnicodeLocaleId(_code_).
           1. Return _code_.
         1. If _type_ is `"region"`, then
           1. If _code_ does not matches the `unicode_region_subtag` production, throw a *RangeError* exceptipon.
```

Comment:
Region (and scripts) subtags should also get canonicalised to replaced outdated subtags with their preferred value.

Reply:
I have concern about this. (canonicalize the region code). There are no pre-defined process in UTS35 for this. The process for the region subtag within unicode_language_id stated in https://unicode-org.github.io/cldr/ldml/tr35.html#Canonical_Unicode_Locale_Identifiers depends on the language code (and script code if present) while there are multiple territories listed in the replacement attribute of territoryAlias.